### PR TITLE
feat(ui): update new-release notifications

### PR DESCRIPTION
**Pull request by huwshimi**
## Done

- Replace the opt-out menu with a settings link on new release notifications.

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- Create a new release notification (instructions are here: https://github.com/canonical-web-and-design/MAAS-squad/issues/2065#issue-669660220).
- Check that the notification has a "See settings" link instead of the opt-out menu.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1507

### CODE REVIEW

File: ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx [hunk 1]
```diff
@@ -46,33 +38,12 @@ const NotificationGroupNotification = ({ id, type }: Props): JSX.Element => {
...
-          }
-          hasToggleIcon
-          position="right"
-          toggleAppearance="base"
-        />
-      )}
+      {showSettings ? (
+        <>
+          {" "}
+          <Link to="/settings/configuration/general">See settings</Link>
```
Comment (anthonydillon) [inline ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx:44]: Shouldn't the link text and to be props of this component?

**Review by anthonydillon — APPROVED** [review]
Code and QA :+1: with a question in the code.

File: ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx (hunk 1 above)
Comment (huwshimi) [reply ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx:44]: This component checks for new release notifications and this link is only shown if it is one. This component is really a wrapper around `<Notification>` with specific use in the top notification list.

Comment (huwshimi) [reply ui/src/app/base/components/NotificationGroup/Notification/Notification.tsx:44]: Hope that made sense :).

**Pull request merged**